backend/tests/character/test_calc_vars.py

```diff
@@ -14,7 +14,7 @@ def test_baseline(self):
         the_shitters_i_raid_with = Comp(
             {Jobs.PLD, Jobs.WAR, Jobs.SCH, Jobs.WHM, Jobs.SAM, Jobs.SAM, Jobs.MCH, Jobs.BLM})
         # have to manually provide pps for testing for now
-        self.assertEqual(20381.368540000003, test_char.calc_damage(200, the_shitters_i_raid_with))
+        self.assertAlmostEqual(20381.368540000003, test_char.calc_damage(200, the_shitters_i_raid_with), places=3)
```

Comment:
given that we're only checking to 3 places, i think the values used for comparison can be truncated at this point

Reply:
fixed